lib/active_merchant/billing/gateways/stripe.rb

```diff
@@ -365,6 +367,21 @@ def add_address(post, options)
         end
       end
 
+      def add_statement_address(post, options)
+        statement_address = options[:statement_address]
+
+        return unless statement_address && statement_address[:address1] &&
+          statement_address[:city] &&
+          statement_address[:zip] &&
+          statement_address[:state]
+        post[:statement_address] = {}
+        post[:statement_address][:line1] = statement_address[:address1]
+        post[:statement_address][:line2] = statement_address[:address2]
+        post[:statement_address][:city] = statement_address[:city]
+        post[:statement_address][:postal_code] = statement_address[:zip]
```

Comment:
it could. I chose zip instead of postal code to stay consistent with the billing address parameters that get passed in but if we want the values to match with the ones stripe accepts thats fine too. I guess it depends on what we were thinking of passing in through the options.